$DragonFly: src/gnu/usr.bin/diff/diff.h.patch,v 1.1 2007/08/05 05:47:00 pavalos Exp $
--- diff.h.orig	2007-08-03 00:26:58.000000000 -0400
+++ diff.h	2007-08-03 00:27:17.000000000 -0400
@@ -21,7 +21,7 @@
    59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.  */
 
 #include "system.h"
-#include <regex.h>
+#include <gnuregex.h>
 #include <stdio.h>
 #include <unlocked-io.h>
 
